bundles/persistence/org.openhab.persistence.jdbc/java/org/openhab/persistence/jdbc/internal/JdbcConfiguration.java

```diff
@@ -0,0 +1,321 @@
+		}
+
+		String rt = (String) configuration.get("rebuildTableNames");
+		if (StringUtils.isNotBlank(rt)) {
+			rebuildTableNames = "true".equals(rt) ? Boolean.parseBoolean(rt) : false;
+			logger.debug("JDBC::updateConfig: rebuildTableNames={}", rebuildTableNames);
+		}
+		// undocumented
+		String ac = (String) configuration.get("maximumPoolSize");
+		if (StringUtils.isNotBlank(ac)) {
+			dBDAO.databaseProps.setProperty("maximumPoolSize", ac);
+		}
+		// undocumented
+		String ic = (String) configuration.get("minimumIdle");
+		if (StringUtils.isNotBlank(ic)) {
+			dBDAO.databaseProps.setProperty("minimumIdle", ic);
+		}
+		// undocumented
+		String it = (String) configuration.get("idleTimeout");
+		if (StringUtils.isNotBlank(it)) {
+			dBDAO.databaseProps.setProperty("idleTimeout", it);
+		}
+		// undocumented
+		String ent = (String) configuration.get("enableLogTime");
+		if (StringUtils.isNotBlank(ent)) {
+			enableLogTime = "true".equals(ent) ? Boolean.parseBoolean(ent) : false;
+		}
+		logger.debug("JDBC::updateConfig: enableLogTime {}", enableLogTime);
+
+		// undocumented
+		String fd = (String) configuration.get("driverClassName");
+		if (StringUtils.isNotBlank(fd)) {
+			dBDAO.databaseProps.setProperty("driverClassName", fd);
+		}
+		// undocumented
+		String ds = (String) configuration.get("dataSourceClassName");
+		if (StringUtils.isNotBlank(ds)) {
+			dBDAO.databaseProps.setProperty("dataSourceClassName", ds);
+		}
+
+		
+		String warn = ""
+				+ "\n\n\t!!!\n\tTo avoid this error, place a appropriate JDBC driver file for serviceName '{}' in addons directory.\n"
+				+ "\tCopy missing JDBC-Driver-jar to your OpenHab/addons Folder.\n\t!!!\n"
+				+ "\tDOWNLOAD: \n";
+		if(serviceName.equals("derby"))           warn += "\tDerby:     Tested version: derby-10.11.1.1.jar               http://mvnrepository.com/artifact/org.apache.derby/derby\n";
+		else if(serviceName.equals("h2"))         warn += "\tH2:        Tested version: h2-1.4.189.jar                    http://mvnrepository.com/artifact/com.h2database/h2\n";
+		else if(serviceName.equals("hsqldb"))     warn += "\tHSQLDB:    Tested version: hsqldb-2.3.3.jar                  http://mvnrepository.com/artifact/org.hsqldb/hsqldb\n";
+		else if(serviceName.equals("mariadb"))    warn += "\tMariaDB:   Tested version: mariadb-java-client-1.2.0.jar     http://mvnrepository.com/artifact/org.mariadb.jdbc/mariadb-java-client\n";
+		else if(serviceName.equals("mysql"))      warn += "\tMySQL      Tested version: mysql-connector-java-5.1.36.jar   http://mvnrepository.com/artifact/mysql/mysql-connector-java\n";
+		else if(serviceName.equals("postgresql")) warn += "\tPostgreSQL:Tested version: postgresql-9.4-1201-jdbc41.jar    http://mvnrepository.com/artifact/org.postgresql/postgresql\n";
+		else if(serviceName.equals("sqlite"))     warn += "\tSQLite:    Tested version: sqlite-jdbc-3.8.11.1.jar          http://mvnrepository.com/artifact/org.xerial/sqlite-jdbc\n";
+		
```

Comment:
Maybe put the exact tested library versions on the wiki and just write "check wiki for versions" here? Else, you would have to remember to update those warning messages every time you update a library.